pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
```

Comment:
```suggestion
- Start an interactive `yacas` session:
```
I think `console` is a little to unspecific. `yacas` isn't like the console you're running it in.